stackslib/src/chainstate/burn/db/sortdb.rs

```diff
@@ -4887,6 +5024,17 @@ impl SortitionDB {
         })
     }
 
+    /// Get the block-commit for a Nakamoto block, given the block-commit's sortition's consensus
+    /// hash and its given last_tenure_id
+    pub fn get_block_commit_for_nakamoto_block(
```

Comment:
Can you expand on the rustdocs for this function with something like:

"If the winning block commit of the sortition identified by `consensus_hash` committed to `last_tenure_id`, then return `Some(winning_block_commit)`, otherwise, returns `None`."

Otherwise, it's not obvious that this returns the _winning block commit_ (and only under certain conditions).

Reply:
This code is dead, actually.  I'll just remove it.